synapse/lib/ast.py

```diff
@@ -3060,6 +3066,10 @@ async def getPropAndValu(self, runt, path):
                                                     name=name, form=path.node.form.name))
 
             valu = path.node.get(name)
+            if isinstance(valu, dict):
+                # dicts get special cased because changing the dict affects the node
+                # while it's in the pipeline but the modification doesn't get stored
+                valu = s_msgpack.deepcopy(valu, use_list=True)
```

Comment:
the default `use_list=False` is probably correct here since we don't want anyone to think they can mutate the returned value?

Reply:
Hmm. Actually, once a copy of the value is returned, the caller should be free to do what they want with it since it won't affect the property value at all, right?

Reply:
A tuple can contain a nested list or dict :)